src/redux/features/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    AddBook: (state, { payload }) => {
+      state.books.push({ payload });
+    },
+    RemoveBook: (state, action) => {
+      const itemId = action.payload;
```

Comment:
- [ OPTIONAL ] I think the previous code reviewer doesn't mention that but still very important to do it so I think to have a unique ID it should assign it to be unique so you can add something like this

> `id: Math.floor(Math.random() * 100),`

Reply:
Hello @Koshao-sys yes I think it is good option you can use whatever you want to have a unique ID 🤩💯